### DISCUSSION TITLE
Need help to get data from syslog server to Wazuh server dash board.

### DISCUSSION THREAD

**Discussion by sandeepittiam** [wazuh/wazuh#22485] [Q&A]
Hi,
   I have configured syslog server and Wazuh server managet both on different machines. Mounted syslog server log file on Wazuh machine and added 
    <localfile>
    <log_format>full_command</log_format>
    <location>/root/mount/firewall.log</location>
  </localfile>
lines on the ossec.conf file as well. Still i am unable to get logs on the dash board. Please let me know if any thing else i need to do.

**Comment by winhigh** [comment]
what exactly you want to do @sandeepittiam  ? are you trying to push logs of wazuh to your server ?
